### ISSUE TITLE
Demo > Result is broken: https://foliotek.github.io/Croppie/

### ISSUE THREAD

**Issue by mariusa** [Foliotek/Croppie#689]
## Expected Behavior
Have the cropped image show when clicking Get Result

## Actual Behavior
Error in browser console:
sweetalert.min.js:1 Failed to load resource: the server responded with a status of 404 

## Steps to Reproduce the Problem

  1. https://foliotek.github.io/Croppie/
  2. Get result

**Comment by kennblvnp** [comment]
Yeah the result isn't doing anything, I think this is major error

**Comment by markusthiel** [comment]
Sweetalert is missing. And EXIF.js too. The result should appear in an "alert" window.
https://lipis.github.io/bootstrap-sweetalert/
